float64[16] EE_T_K
---
bool success
string error

